Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/vending_machines.yml

```
@@ -263,7 +263,10 @@
     CMJumpsuitMP: 2
     CMHeadBeretRed: 6
     CMGlassesSecurity: 3
-    CMHeadset: 6
+    RMCScrubsPrisonMedsec: 4
+    RMCScrubsPrisonMaxsec: 2
+    RMCHeadsetPrisoner: 6
+    RMCShoesPrisoner: 6
```

These don't go into the sectech vendor, they go into closets
The headset is also intended to be here
```suggestion
    CMHeadset: 6
```